tasks/main.yml

```diff
@@ -138,6 +147,17 @@
     state: absent
   when: bootloader_remove_password | bool
 
+- name: Ensure that the default kernel remains the default
```

Comment:
@martinpitt So the role can do this. But after reading thoroughly your discussion in https://bugzilla.redhat.com/show_bug.cgi?id=2361624, I realize that the default grubby behavior is the opposite - to make the new added kernel default.
I think the best would be to remove this workaround and fix the text on Fedora 42. WDYT?

Reply:
I think what you suggest is a hack. And even though this hack would make the role work, it might hide the actual issue that might appear somewhere else and would be harder to troubleshoot.
I'd say that now that role is able to configure the default kernel with this PR being merged, users will be able to use the role to fix this bug/feature of grub/Fedora.

Reply:
I was thinking if it's better to ignore this bug in the role completely, just don't touch anything. I am afrad that we can make someone's processes if we add such a behavior to the role.

Reply:
@richm what do you think?

Reply:
> I was thinking if it's better to ignore this bug in the role completely,

I agree.

The feature that allows the user to set the default kernel should be sufficient for those cases where the user wants to ensure that a particular kernel is booted.

We should document that `grubby` can change the default kernel due to bug xxx, and that if the user must ensure that a particular kernel is booted, they can use this feature to set the default kernel.